editor/components/default-block-appender/test/index.js

```diff
@@ -14,16 +14,22 @@ describe( 'DefaultBlockAppender', () => {
 		expect( onAppend ).toHaveBeenCalledWith();
 	};
 
+	it( 'should render nothing if not visible', () => {
+		const wrapper = shallow( <DefaultBlockAppender /> );
+
+		expect( wrapper.type() ).toBe( null );
+	} );
+
 	it( 'should match snapshot', () => {
 		const onAppend = jest.fn();
-		const wrapper = shallow( <DefaultBlockAppender onAppend={ onAppend } /> );
+		const wrapper = shallow( <DefaultBlockAppender isVisible onAppend={ onAppend } showPrompt /> );
```

Comment:
Should we set defaults instead of providing `isVisible` and `showPrompt ` props here?

Reply:
It seems like `false` are the defaults, skip it :)